src/Roslyn.Suppressions/Roslyn.Suppressions.UnitTests/SuppressionCheckerTests.cs

```diff
@@ -104,7 +106,7 @@ public void IsSuppressed_HasIssues_NoMatches_ReturnsFalse()
 
             var suppressedIssues = new SuppressedIssue[]
             {
-                CreateIssue("S111", "c:\\wrongFile1.txt")
+                CreateIssue("S111", "wrongFile1.txt")
```

Comment:
The product code change is simple. However, the existing tests needed a bit more fixing up because the product code is now stricter and contains multiple assertions about the input values i.e. the local path should be absolute and the server issue path should be relative.

The main required change was to update the server paths set in the tests to be relative.